client/src/test-runner/adapter.ts

```diff
@@ -21,7 +21,20 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 */
+import { Test } from "mocha";
```

Comment:
Is this the right import? My IDE also seems to think that a lot of these are unused.